apps/emqx/src/emqx_limiter/etc/emqx_limiter.conf

```diff
@@ -0,0 +1,11 @@
+limiter {
```

Comment:
nit: Will these section be generated into final `emqx.conf`?

Does this meet our need for a simplified configuration? @zmstone @zhongwencool